src/main/scala/com/codacy/duplication/pmd/Cpd.scala

```diff
@@ -98,31 +109,38 @@ object Cpd extends DuplicationTool {
           ScalaTokenizer,
           cpdScala.getExtensions.asScala: _*) {}
         Some(cpdConfiguration(scalaLanguage, 50, options))
-      case Languages.CSharp => Some(cpdConfiguration(new CsLanguage, 50, options))
-      case _                => None
+      case _ => None
     }
   }
 
-  private def cpdConfiguration(cpdLanguage: CPDLanguage,
-                               defaultMinToken: Int,
-                               options: Map[DuplicationConfiguration.Key, DuplicationConfiguration.Value]) = {
+  // TODO: Move to codacy-plugins-api
```

Comment:
codacy-plugins-api does not have dependencies, it only has case classes and minor functionality for them. Json enconding is not suposed to be there.

Reply:
How about here: https://github.com/codacy/codacy-duplication-scala-seed/blob/master/src/main/scala/com/codacy/plugins/api/package.scala